Sources/ProjectSpec/TargetSource.swift

```diff
@@ -10,20 +11,38 @@ public struct TargetSource {
     public var excludes: [String]
     public var type: SourceType?
     public var optional: Bool
+    public var buildPhase: BuildPhase?
+
+    public enum BuildPhase: String {
+        case sources
+        case headers
+        case resources
+        case none
+
+        public var buildPhase: xcproj.BuildPhase? {
+            switch self {
+            case .sources: return .sources
+            case .headers: return .headers
+            case .resources: return .resources
+            case .none: return nil
+            }
+        }
+    }
 
     public enum SourceType: String {
         case group
         case file
         case folder
     }
 
-    public init(path: String, name: String? = nil, compilerFlags: [String] = [], excludes: [String] = [], type: SourceType? = nil, optional: Bool = false) {
+    public init(path: String, name: String? = nil, compilerFlags: [String] = [], excludes: [String] = [], type: SourceType? = nil, optional: Bool = false, buildPhase: BuildPhase? = nil) {
```

Comment:
I think @toshi0383 mentioned something in another PR but what do you think about splitting this into multiple lines to make it easier to read. Out of curiosity, is there any linter being passed on CI in every PR? That could help keeping method names short.

Reply:
At the moment no linter is run on CI. There's a `format_code` make task though that I occasionally run. This is yet to be automated. Along with the automation, rules for line wrapping can be added 👍